### ISSUE TITLE
Block REST request on backend

### ISSUE THREAD

**Issue by glebfeklistov** [tomavic/cordova-apple-wallet#26]
Hello. We trying to get payload from our backend, about 50% happening application catched request and stoped them until we pressed cancel in wallet and send its when we returned in application. There is my code:

```js
export default function(props) {
  const { card, wallet, provider, last4Fpan } = props;

  return Apay.getCardCerts(last4Fpan, provider).then(certs => {
    const data = {
        nonce: certs.data.nonce,
        nonceSignature: certs.data.nonceSignature,
        certificates: [certs.data.certificateLeaf, certs.data.certificateSubCA],
    };

    // this request blocked
    Api.post(`http://myserver.com/echo`, {data}).then(payload => {
      console.log('ECHO', payload);
    });

    // and this request blocked
    return Api.postSync(`v1/wallets/addCard`, data).then(({ payload }) => {
      console.log('PAYLOAD', payload);
      return Apay.sendCardCerts(JSON.parse(payload));
    });
  });
}
```

It`s happened randomly 10% of request or 100% request blocked same time. What happend? Any ideas? Are you has some issues?

**Comment by tomavic** [comment]
@glebfeklistov Why did you closed the issue?

**Comment by glebfeklistov** [comment]
Request blocked by cordova-plugin-wkwebview-engine, i will create another one at monday if not done another issue. Ty.

**Comment by JohnathanWhite** [comment]
@glebfeklistov We are experiencing the same issue. What in specific with wkwebview was blocking the requests? Thank you!

**Comment by glebfeklistov** [comment]
@JohnathanWhite Problem not in wkwebview, problem in this plugin, I don't remember where was problem, I just wrote another plugin for my project

**Comment by faugusztin** [comment]
In my experience, the issue came down to JS<->Cordova bridge stopping communicating in the process of adding payment card. The solution in my case was to move the whole backend call into native code with most of the solution hardcoded, but for an universal solution, one would have to add request/response templating, passing request headers with authorization/content types and so on. 

Note how Ionic Enteprise Apple Payment Pass API also has templating etc, they probably found out about it too:
https://ionic.io/docs/premier-plugins/apple-payment-pass/#card-addition

**Comment by tomavic** [comment]
@glebfeklistov Hello

If there is a suggestion to make this plugin completely useful I would appreciate that.

**Comment by tomavic** [comment]
@faugusztin Hi

I found it interesting that I make something for free, and ionic team make it premium and use it to make some money. If you have a suggestion on how to improve the things up please contact me or make Pull Request as you did before. I appreciate your efforts.
